components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/base/base_container.html

```diff
@@ -195,6 +195,61 @@
         {% if page_size %}
             var PAGE_SIZE = {{ page_size }};
         {% endif %}
+
+
+        // ** "Open With" config used by E.g. ome.tree.js **
+        WEBCLIENT.OPEN_WITH = [];
+        {% if ome.open_with %}
+            // This is a json dump, so we can add it directly without escaping
+            WEBCLIENT.OPEN_WITH = {{ ome.open_with|safe }};
+        {% endif %}
+        // Helper can be used by 'open with' plugins to add isEnabled()
+        // handlers to the OPEN_WITH object.
+        OME.setOpenWithEnabledHandler = function(label, fn) {
+            // look for label in OPEN_WITH
+            WEBCLIENT.OPEN_WITH.forEach(function(ow){
+                if (ow.label === label) {
+                    ow.isEnabled = function() {
+                        // wrap fn with try/catch, return true by default
+                        var args = Array.from(arguments);
+                        try {
+                            enabled = fn(args);
+                        } catch (e) {
+                            console.log("Open with " + label + ": " + e);
```

Comment:
Certainly for normal use we wouldn't want to ```console.log()``` but in this case we want to give some feedback for why the plugin is not working. Without a try/catch you'd see any error in the console log anyway, but we need to be more defensive here since an error would otherwise break the whole jsTree menu.